build.gradle

```diff
@@ -47,4 +47,6 @@ dependencies {
     testCompile 'io.cucumber:cucumber-java:4.3.1'
 // https://mvnrepository.com/artifact/org.testng/testng
     testCompile group: 'org.testng', name: 'testng', version: '7.0.0'
+    compile group: 'net.masterthought', name: 'cucumber-reporting', version: '4.6.0'
+    compile group: 'io.cucumber', name: 'cucumber-testng', version: '4.7.1'
```

Comment:
I think that you use only one type and version of cucumber-tesng and cucumber-java E.g.
**testCompile** group: 'io.cucumber', name: 'cucumber-java', version: **'4.7.1'**
**testCompile** group: 'io.cucumber', name: 'cucumber-testng', version: **'4.7.1'**

Reply:
done